configurations/memory_basic.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Hast du die Config gewollt geändert oder ist die in einen Commit gerutscht?

Reply:
Teils Teils. Die `"type": "ReacherMemoryAttr",` Zeile ist wichtig, der Rest nicht